packages/design-tokens/src/tokens/color/background.ts

```diff
@@ -11,10 +11,10 @@ export const background = {
 	danger: colorPaletteRed50,
 	primary: colorPaletteWhite,
 	secondary: colorPaletteGrey100,
-	sideNav: '#021c6b',
+	'side-nav': '#021c6b',
 	success: colorPaletteGreen50,
 	surface: colorPaletteWhite,
 	tertiary: colorPaletteGrey200,
-	topNav: '#1f3987',
+	'top-nav': '#1f3987',
```

Comment:
camel-casing introduced a breaking change, insuring that these are kebab-case to use within the theme.

Reply:
actually going to revert this and handle it in the them